src/lightning_app/components/serve/python_server.py

```diff
@@ -16,12 +16,12 @@
 
 logger = Logger(__name__)
 
-__doctest_skip__ = ["PythonServer", "PythonServer.*"]
-
-
 # Skip doctests if requirements aren't available
-if not _is_torch_available():
-    __doctest_skip__ += ["PythonServer", "PythonServer.*"]
+if not module_available("lightning_api_access"):
+    __doctest_skip__ = ["PythonServer", "PythonServer.*"]
+# Skip doctests if requirements aren't available
+elif not _is_torch_available():
+    __doctest_skip__ = ["PythonServer", "PythonServer.*"]
```

Comment:
The comment is duplicated, also this can be merged

```suggestion
if not module_available("lightning_api_access") or not _is_torch_available():
    __doctest_skip__ = ["PythonServer", "PythonServer.*"]
```